web-client/src/models/requests/index.ts

```diff
@@ -285,7 +328,9 @@ export class Request implements IRequest {
   }
 
   public static factory(data: IRequest): Request {
+    console.log('factoryx', data.pinUserSnapshot);
     return new Request(
```

Comment:
This is bad code and should be removed.  I later found out that id is used to refer to a numeric key that also corresponds to the row of the item.